src/main/java/chess/controller/ChessControllerAdvice.java

```diff
@@ -0,0 +1,14 @@
+package chess.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class ChessControllerAdvice {
```

Comment:
```suggestion
@RestControllerAdvice
public class ChessControllerAdvice {
```
API 형태로 반환을 예상했다면, RestController가 자연스러울 것 같네요

Reply:
그렇네요! 수정하였습니다~!